app/assets/javascripts/components/reviews/review_form.js.jsx

```diff
@@ -0,0 +1,27 @@
+var ReviewForm = React.createClass({
+
+  getInitialState: function(){
+    return { film: this.props.film, content: ''}
```

Comment:
I would recommend initializing `film` with an empty array because the rest of the code expects and array and if nothing is passed in it will be undefined and break the app.